Tests/test_tcons.py

```diff
@@ -0,0 +1,143 @@
+import sys
```

Comment:
Dude, everything needs to be cons cons cons
is us

Reply:
Done, dude
Everything is now cons cons cons 
Bro!